packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -79,6 +79,8 @@ export interface EngineReportingOptions {
   // 'sendReport()' on other signals if you'd like. Note that 'sendReport()'
   // does not run synchronously so it cannot work usefully in an 'exit' handler.
   handleSignals?: boolean;
+  // Disables reporting on an interval for stateless environments
+  disableInterval?: boolean;
```

Comment:
It seems clearer to me to turn this around, so instead of `disableInterval` maybe something like `sendReportsImmediately`.